packages/map-template/src/components/BottomSheet/BottomSheet.jsx

```diff
@@ -37,13 +42,17 @@ function BottomSheet({ currentLocation, setCurrentLocation, currentCategories, o
     const [wayfindingSheetSize, setWayfindingSheetSize] = useState();
     const [searchSheetSize, setSearchSheetSize] = useState();
 
+
     /*
      * React on changes on the current location.
```

Comment:
I guess it reacts on more than current location now 😄

Reply:
✅